src/website/apps/api/v2/serializers/event.py

```
@@ -176,6 +344,56 @@ def get_has_slug(self, obj):
+                pass
+        # Fallback: use prefetched cache or count().
+        cached = getattr(obj, '_prefetched_objects_cache', None)
+        if cached is not None:
+            for entry in cached.values():
+                try:
+                    return len(entry)
+                except TypeError:
+                    break
+        try:
+            return obj.event_organizer_links.count()
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)